app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java

```diff
@@ -364,9 +377,17 @@ public void saveAlert(final models.ebean.Alert alert) {
          * {@inheritDoc}
          */
         @Override
-        public long getEtag() {
-            final SqlQuery sqlQuery = Ebean.createSqlQuery("SELECT CURRVAL('portal.alerts_etag_seq') AS etag;");
-            return sqlQuery.findUnique().getLong("etag");
+        public long getEtag(final Organization organization) {
+            final SqlQuery sqlQuery = Ebean.createSqlQuery("SELECT etag from portal.alerts_etags where organization = ?;");
```

Comment:
I'm not sure I love calling this etags in the database. We're tracking the version of the set of data (e.g. an organization's alerts). Would these be more appropriate in an organization table as a column?